File: TYPESCRIPT PROGRAMMING/Test50.ts
// 50. PRINT PERFECT SQUARE NUMBERS FROM 0 TO N
const PerfectSquareN = (range: number): void => {
    for(let i: number = 0; i * i <= range; i++) {
        process.stdout.write(`${i * i} `);
    }
    console.log();
}
PerfectSquareN(10);
PerfectSquareN(20);
PerfectSquareN(50);
PerfectSquareN(100);

File: JAVA PROGRAMMING/Test50.java
// 50. PRINT PERFECT SQUARE NUMBERS FROM 0 TO N
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range Value: ");
        int range = in.nextInt();

        for(int i = 0; i * i <= range; i++) {
            System.out.print(i * i+" ");
        }

        in.close();
    }
}